src/handwritten/task.py

```diff
@@ -402,6 +402,28 @@ def is_task_done(self):
 
         return is_task_done
 
+    def perform_bridge_offset_nulling_cal(self, channel="", skip_unsupported_channels=False):
```

Comment:
Please add the parameter documentation to the docstring in Google docstring format.

```
Args:
    channel: Whatever

    skip_unsupported_channels: Whatever
```

Reply:
Thanks for pointing this out, I have added the parameter documentation according to Labview HELP documentation.